README.md

```diff
@@ -1,84 +1,210 @@
 # kubeprovenance
 
-A Kubernetes Aggregated API Server to find out Provenance information for different Kuberentes Kinds.
-
+A Kubernetes Aggregated API Server to find out Provenance information for different Kuberentes Custom Resources.
 
 ## What is it?
 
-kubeprovenance is a tool that helps you find Provenance information about different Kubernetes Kinds
-in your cluster. 
-The information obtained from kubeprovenance can be used to determine how your cluster
-has evolved over time (its provenance/lineage). 
-Using this information one can answer questions such as:
-
-- How many deployments have occurred on a cluster?
-
-- What actions have been executed on a cluster?
-
-- How configuration values have changed over time?
-
+kubeprovenance is a tool that helps you find Provenance information about different Kubernetes custom resources
+in your cluster. <br/>
+The information obtained from kubeprovenance can be used to determine how your cluster <br/>
+has evolved over time (its provenance/lineage). <br/>
 
 ## How does it work?
 
-kubeprovenance uses Kubernetes Audit logs to build the provenance information.
+kubeprovenance uses Kubernetes Auditing to build the provenance information.
 
 In building this API server we tried several approaches. You can read about our experience  
 [here](https://medium.com/@cloudark/our-journey-in-building-a-kubernetes-aggregated-api-server-29a4f9c1de22).
 
+## Try it Out: 
+**Steps to Run Kubernetes Local Cluster on a GCE or AWS instance (or any node), configure auditing and running/testing Kubeprovenance aggregated api server** <br/> 
+
+**1. Setting up environment. reference: https://dzone.com/articles/easy-step-by-step-local-kubernetes-source-code-cha** <br/>
+ssh to your VM <br/>
+sudo su - <br/>
+apt-get install -y gcc make socat git<br/>
+
+**2. Install Golang 1.10.3:** <br/>
+wget https://dl.google.com/go/go1.10.3.linux-amd64.tar.gz
+sudo tar -C /usr/local -xzf go1.10.3.linux-amd64.tar.gz
+export PATH=$PATH:/usr/local/go/bin <br/>
+
+**3. Install etcd3.2.18:**
+curl -L https://github.com/coreos/etcd/releases/download/v3.2.18/etcd-v3.2.18-linux-amd64.tar.gz -o etcd-v3.2.18-linux-amd64.tar.gz && tar xzvf etcd-v3.2.18-linux-amd64.tar.gz && /bin/cp -f etcd-v3.2.18-linux-amd64/{etcd,etcdctl} /usr/bin && rm -rf etcd-v3.2.18-linux-amd64* <br/>
+**4. Install Docker**<br/>
+sudo apt-get update <br/>
+sudo apt-get install docker-ce <br/>
+
+set up your go workspace, set the GOPATH to it. this is where all your go code should be. <br/>
+export GOPATH=/gopath <br/>
+
+**5. Get The Kubernetes Source Code:** <br/>
+git clone --depth 1 https://github.com/kubernetes/kubernetes <br/>
```

Comment:
I know that the article mentions to do this.
But line 43 makes line 42 unnecessary imo. I don't know what is achieved by cloning kubernetes repo in two different locations. Line 42 clones it in the home directory. Line 43 clones it in go src.
The one in go src is the one that is needed. So we can get rid of line 42.